presto-main/src/test/java/com/facebook/presto/type/TestRowIdType.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.type;
+
+import com.facebook.presto.common.block.Block;
+import com.facebook.presto.common.block.BlockBuilder;
+import com.facebook.presto.common.type.SqlVarbinary;
+import io.airlift.slice.Slices;
+import org.testng.annotations.Test;
+
+import java.nio.ByteBuffer;
+import java.nio.charset.StandardCharsets;
+
+import static com.facebook.presto.common.type.RowIdType.ROW_ID;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertTrue;
+
+public class TestRowIdType
+        extends AbstractTestType
+{
+    public TestRowIdType()
+    {
+        super(ROW_ID, SqlVarbinary.class, createTestBlock());
+    }
+
+    public static Block createTestBlock()
+    {
+        BlockBuilder blockBuilder = ROW_ID.createBlockBuilder(null, 1);
+        for (int i = 0; i < 10; i++) {
+            ByteBuffer rowId = ByteBuffer.allocate(4 + 100 + 4 + 8 + 8 + 4 + 2)
```

Comment:
IMHO a byte array with random size and content will do just fine. We don't need to be this specific for the unit test.